src/main/java/org/openstreetmap/atlas/checks/flag/CheckFlag.java

```diff
@@ -481,6 +484,20 @@ public void setChallengeName(final String challengeName)
         this.challengeName = challengeName;
     }
 
+    /**
+     * Set the {@link CheckFlag}'s identifier equal to the sorted concatenation of all the ids of
+     * its flagged features.
+     *
+     * @return this {@link CheckFlag}
+     */
+    public CheckFlag setObjectIdentifiersAsFlagIdentifier()
```

Comment:
Can we be sure that the sorted concatenation of 2 or more IDs is not itself the same as some other single feature's ID?

Reply:
We could append the feature type after the identifier if this is a risk, for example as in your unit test the flag would be "1n2n"